1.8/development/index.md

```diff
@@ -1,16 +1,103 @@
 ---
-post_title: Development of DC/OS Services
-nav_title: Development
+post_title: Developing Services
 menu_order: 4
 ---
 
-# <a name="universe"></a> Mesosphere Universe
+This section describes the developer-specific DC/OS components, explaining what is necessary to package and provide your own service on DC/OS.
 
-[Universe][1] is the DC/OS package repository that contains services like Spark, Cassandra, Jenkins, and many others. At their simplest, DC/OS packages are bundled containerized apps that can be deployed onto a cluster. Everyone is welcome and encouraged to submit a package to Universe. Some packages are "selected" by Mesosphere and are identified as such in the DC/OS UI and DC/OS CLI.
+The Mesosphere Datacenter Operating System (DC/OS) provides the optimal user experience possible for orchestrating and managing a datacenter. If you are an Apache Mesos developer, you are already familiar with developing a framework. DC/OS extends Apache Mesos by including a web interface for health checks and monitoring, a command-line, a service packaging description, and a [repository](/docs/1.8/usage/repo/) that catalogs those packages.
 
-All services in the Universe are required to meet a standard as defined by the DC/OS project team. For details on publishing a release of a DC/OS service, see [Publish a Package][2] in the Universe documentation.
+# <a name="universe"></a>Package Repositories
 
-For detailed information about the JSON files of a package, see the [Universe][1] docs.
+The DC/OS Universe contains all of the services that are installable DC/OS. For more information on DC/OS Universe, see the [GitHub Universe repository(https://github.com/mesosphere/universe).
 
- [1]: http://mesosphere.github.io/universe/
- [2]: http://mesosphere.github.io/universe/#publish-a-package-1
+All services in the package repositories are required to meet a certain standard as defined by Mesosphere. For details on submitting a DC/OS service, see [Contributing a package](http://mesosphere.github.io/universe/#publish-a-package-1).
```

Comment:
You could avoid referencing repositories altogether, and just go with "All packaged services are required..." or something similar.